src/Compiler/AbstractIL/ilwrite.fs

```diff
@@ -4122,7 +4122,7 @@ let writeBinaryAux (stream: Stream, options: options, modul, normalizeAssemblyRe
           | Some AMD64 -> writeInt32AsUInt16 os 0x8664      // Machine - IMAGE_FILE_MACHINE_AMD64
           | Some IA64 -> writeInt32AsUInt16 os 0x200        // Machine - IMAGE_FILE_MACHINE_IA64
           | Some ARM64 -> writeInt32AsUInt16 os 0xaa64      // Machine - IMAGE_FILE_MACHINE_ARM64
-          | Some ARM -> writeInt32AsUInt16 os 0x1c0         // Machine - IMAGE_FILE_MACHINE_ARM
+          | Some ARM -> writeInt32AsUInt16 os 0x1c4         // Machine - IMAGE_FILE_MACHINE_ARM
```

Comment:
I don't really know anything about this, but according to [this table](https://docs.microsoft.com/en-us/windows/win32/debug/pe-format#machine-types) this value is called `IMAGE_FILE_MACHINE_ARMNT`. So maybe update the comment to avoid possible future confusion :)